edge/pkg/edged/edged_pods.go

```diff
@@ -1244,15 +1244,22 @@ func (e *edged) ServeLogs(w http.ResponseWriter, req *http.Request) {
 }
 
 func (e *edged) GetExec(podFullName string, podUID types.UID, containerName string, cmd []string, streamOpts remotecommand.Options) (*url.URL, error) {
-	return nil, nil
+	container, err := e.findContainer(podFullName, podUID, containerName)
+	if err != nil {
+		return nil, err
+	}
+	if container == nil {
+		return nil, fmt.Errorf("container not found (%q)", containerName)
+	}
+	return e.streamingRuntime.GetExec(container.ID, cmd, streamOpts.Stdin, streamOpts.Stdout, streamOpts.Stderr, streamOpts.TTY)
 }
 
 func (e *edged) GetAttach(podFullName string, podUID types.UID, containerName string, streamOpts remotecommand.Options) (*url.URL, error) {
-	return nil, nil
+	panic("GetAttach need to be continued")
```

Comment:
nit: maybe simply "GetAttach need to be implemented" ?

Reply:
ok